pages/linux/ceph.md

```diff
@@ -0,0 +1,32 @@
+# ceph
+
+> Ceph is a free-software storage platform, implements object storage on a single distributed computer cluster, and provides interfaces for object, block and file-level storage.
+> More information: <https://ceph.io>.
+
+- Check or watch cluster health:
+
+`ceph {{status | -w}}`
+
+- Check cluster usage stats:
+
+`ceph df`
+
+- Get the statistics for the placement groups in your cluster:
+
+`ceph pg dump [--format {{plain | json}}]`
+
+- Create a storage pool:
+
+`ceph osd pool create {pool_name} {page_number}`
+
+- Delete a storage pool:
+
+`ceph osd pool delete {pool_name} [--yes-i-really-really-mean-it]`
+
+- Rename a storage pool:
+
+`ceph osd pool rename {current-pool-name} {new-pool-name}`
+
+- Self-repairing pool storage:
```

Comment:
Does this read better? I feel like we should always use verbs to describe what a command does.

```suggestion
- Self-repair pool storage:
```

Reply:
I agree.